modules/tinymce/src/plugins/image/test/ts/browser/DialogUpdateTest.ts

```diff
@@ -3,13 +3,24 @@ import { UnitTest } from '@ephox/bedrock-client';
 import { ApiChains, TinyLoader } from '@ephox/mcagar';
 import Plugin from 'tinymce/plugins/image/Plugin';
 import SilverTheme from 'tinymce/themes/silver/Theme';
-import { cAssertCleanHtml, cAssertInputValue, cFillActiveDialog, cSubmitDialog, cWaitForDialog, generalTabSelectors } from '../module/Helpers';
+import {
```

Comment:
Don't chop imports please, they should wrap when they get too long instead (there isn't a linter rule for this unfortunately).

- [ ] Fix imports

Reply:
ok, looks horrible to me :-) but done.